### DISCUSSION TITLE
Displaying task number/id in displays

### DISCUSSION THREAD

**Discussion by Shoshana** [Leantime/leantime#734] [General]
Is there any way of displaying the task number/id in the views?

Our team uses the ids as reference all the time and although it's in the URL and is searchable, it would be great if it was more visible.

Thanks!

**Comment by gloriafolaron** [comment]
@Shoshana when you go to look for the id currently, any thoughts on where you and your team would naturally go to look / expect to see it?

**Reply by Shoshana** [reply]
We would expect to see it next to the title on the kanban or table view. That's where it matters most - on the details tab perhaps above the T-do name although that is less important. Thank you!

**Reply by gloriafolaron** [reply]
@marcelfolaron just want to call this out as we look at the new updates planning to go in.

**Comment by axelrindle** [comment]
This feature was shipped in [v2.2.0](https://github.com/Leantime/leantime/releases/tag/v2.2.0).

[Here](https://github.com/Leantime/leantime/pull/703) is the related pull request.
